src/cadenza.js

```diff
@@ -432,6 +432,19 @@ export class CadenzaClient {
     return this.#show(resolvePath(backgroundMapView), params, signal);
   }
 
+  /**
+   * Set the visibility of a layer in an embedded map view. The map view must be shown prior to calling this.
+   *
+   * @param {WorkbookLayerPath} layer - The layer to show or hide, identified by its path
+   * @param {boolean} visible - The visibility state of the layer
+   */
+  setLayerVisibility(layer, visible = true) {
+    this.#log('CadenzaClient#setLayerVisibility', ...arguments);
+    if (layer) {
```

Comment:
This if() is not needed, the type is not nullable.

Reply:
ok